packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -541,6 +554,11 @@ export function messageChannelMiddleware(
             channel.postMessage(action)
             break
 
+          case ActionTypes.MOUNT_COMPONENT:
+          case ActionTypes.UNMOUNT_COMPONENT:
+            channel.postMessage(action, undefined, { lifecycle: Lifecycle.AFTER_INITIALIZED })
```

Comment:
Once we have a state machine that defines the "correct" message order, I wonder if we can keep the `postMessage` simple and automatically dispatch buffered messages in the right order on `initialize` behind the scene.